app/controllers/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to new_user_path
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+
+    if @user.update(user_params)
+      redirect_to @user
```

Comment:
[REQUIRED] Please check this line because it causes an error when I update user:
![1](https://user-images.githubusercontent.com/58552748/107484025-e5384700-6b92-11eb-9b91-39af84a1a9dd.png)